lib/matplotlib/font_manager.py

```diff
@@ -877,9 +877,8 @@ def set_math_fontfamily(self, fontfamily):
         fontfamily : str
             The name of the font family.
 
-            Available font families are defined in the
-            lib/matplotlib/mpl-data/matplotlibrc file
-            :ref:`here <customizing-with-matplotlibrc-files>`
+            See
+            :ref:`available font families <customizing-with-matplotlibrc-files>`.
```

Comment:
I think this description would now make the user expect the link to take them straight to a list of font families.  In fact, they have to scroll down and search for it in the matplotlibrc comments.  So I think the original was mostly right.  Maybe something like “Available font families are defined in the default matplotlibrc file”?